bin/hbase-config.cmd

```diff
@@ -37,7 +37,7 @@ if "%1" == "--help" (
   echo Error usage! You should use it like 'hbase --help' or 'hbase -h';
   exit /B 2
 )
-if "%1" == "--help" (
+if "%1" == "-h" (
```

Comment:
Oh wait, let's combine both of these if conditions.
Something like:
```
if ["%1" == "-h"] || ["%1" == "--help"]
```

Reply:
I am not aware of exact syntax, but with `or` condition for `--help` and `-h`, we should be good with this PR.
Thanks